src/components/accordion-item/accordion-item.tsx

```diff
@@ -43,23 +43,26 @@ export class AccordionItem implements ConditionalSlotComponent {
   //
   //--------------------------------------------------------------------------
 
+  /** Specifies a description for the component. */
+  @Prop() description: string;
+
   /** When `true`, the component is expanded. */
   @Prop({ reflect: true, mutable: true }) expanded = false;
 
   /** Specifies heading text for the component. */
   @Prop() heading: string;
 
-  /** Specifies a description for the component. */
-  @Prop() description: string;
+  /** When `true`, the icon will be flipped when the element direction is right-to-left (`"rtl"`). */
+  @Prop({ reflect: true }) iconFlipRtl: FlipContext;
 
   /** Specifies an icon to display at the start of the component. */
   @Prop({ reflect: true }) iconStart: string;
 
   /** Specifies an icon to display at the end of the component. */
   @Prop({ reflect: true }) iconEnd: string;
 
-  /** When `true`, the icon will be flipped when the element direction is right-to-left (`"rtl"`). */
-  @Prop({ reflect: true }) iconFlipRtl: FlipContext;
+  /** Specifies the size of the component inherited from the `accordion`. */
+  @Prop({ reflect: true }) scale: Scale = "m";
```

Comment:
If this is not meant to be public, can we add an `* @internal` tag so it is excluded from documentation?